flameshot.pro

```diff
@@ -270,7 +270,7 @@ unix:!macx {
     }
 
     INSTALLS += target \
-        icon \
+        appsvgicon \
```

Comment:
Why not simply `iconsvg`? I think is more descriptive that way.

Maybe `icon` can be used for a 48x48 PNG image.

Reply:
If there's another icon in the future but it is not an app icon, say it has to be put into `/$theme_name/scalable/actions/` dir, the `iconsvg` target is not applicable.

I'm not really familiar with `qmake`; is there an option to install the whole directory? In that way we may generate directory tree in advance and install all files in one target.

Reply:
That would be better (install whole directory)

I think using a wildcard would allow you to install a whole directory, but I'm not sure, I'll try.